app/models/user.rb

```diff
@@ -7,4 +7,5 @@ class User < ApplicationRecord
   mount_uploader :profile_photo, ProfilePhotoUploader
   validates :username, uniqueness: true
   validates :username, presence: true
+  has_many :reviews
```

Comment:
For all of your models, please follow this order:
Associations
Validations
Uploader
Custom methods (if applicable)

And remember to leave spaces between them all.